website/content/labs/lab1.md

```diff
@@ -451,8 +468,7 @@ You will be graded partially on discovering the identity of command X, and parti
 
 ### Submitting your lab
 
-Before submitting, make sure you've committed and pushed your changes to the `tasks.txt` file (remember you can run `git status` to check this). Make sure you've set up the `chisubmit` tool as described in [How to submit your labs]({{< relref "submit.md" >}}), and then run the following:
+Before submitting, make sure you've committed and pushed your changes to the `tasks.txt` file (remember you can run `git status` to check this). 
 
-    chisubmit student assignment register lab1
-    chisubmit student assignment submit lab1
+TODO: Gradescope instructions
```

Comment:
no TODOs on `master`! 😡

Reply:
That's why this PR hasn't been merged yet :-P